asmcomp/amd64/emit.mlp

```diff
@@ -171,7 +171,18 @@ let emit_label lbl =
 
 let label s = sym (emit_label s)
 
-let def_label s = D.label (emit_label s)
+module Int = Numbers.Int
```

Comment:
Let's move this "module" declaration up to the top of file file.

Reply:
Done. I had to add a new function Numbers.Int.to_string because of a preceding reference to Int.to_string.